dts/common/freq.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DT_FREQ_H
+#define __DT_FREQ_H
+
+#define DT_FREQ_K(x) ((x) * 1000)
+#define DT_FREQ_M(x) ((x) * 1000 * 1000)
```

Comment:
Why not use existing macros `KHZ()` and `MHZ()` from `include/sys/util.h`?

Reply:
My understanding is that dts files should remain independent from zephyr code.

Reply:
Ok, makes sense.

I would probably prefer to have the entire unit (e.g. `DT_FREQ_KHZ` or just `DT_KHZ`) in the name, but won't insist.

Reply:
> My understanding is that dts files should remain independent from zephyr code.

util_macro.h is available from DTS; see dt-util.h. Perhaps we need to move some definitions into it?